### PR TITLE
[TR] translations of week 12 & review of translations of week 6

### PR DESCRIPTION
**Pull request by mevah**

### CODE REVIEW

File: docs/tr/week12/12-3.md [hunk 1]
```diff
@@ -0,0 +1,579 @@
+---
+lang-ref: ch.12-3
```
Comment (Atcold) [inline docs/tr/week12/12-3.md:2]: `lang: tr` is missing.

**Review by Atcold — CHANGES_REQUESTED** [review]
You need to update `docs/_config.yml`.

There are two files, the first one is in the wrong directory.
image

**Comment by mevah** [comment]
sorry, made changes.

**Review by Atcold — APPROVED** [review]
Fantastic! Thanks!

**Pull request merged**